### PR TITLE
Allow read operations during namespace handover

### PR DESCRIPTION
**Pull request by yux0**
## What changed?
Allow read operations during namespace handover

## Why?
Allow read operations during namespace handover. Read ops should not impact handover.

### CODE REVIEW

File: common/rpc/interceptor/namespace_validator.go [hunk 1]
```diff
@@ -71,10 +71,30 @@ var (
 	defaultAllowedNamespaceStates = []enumspb.NamespaceState{enumspb.NAMESPACE_STATE_REGISTERED, enumspb.NAMESPACE_STATE_DEPRECATED}
 
 	allowedMethodsDuringHandover = map[string]struct{}{
-		"UpdateNamespace":                  {},
-		"GetReplicationMessages":           {},
-		"ReplicateEventsV2":                {},
-		"GetWorkflowExecutionRawHistoryV2": {},
+		"DescribeNamespace":                  {},
+		"UpdateNamespace":                    {},
+		"GetReplicationMessages":             {},
```
Comment (xwduan) [inline common/rpc/interceptor/namespace_validator.go:76]: How about streaming api?

File: common/rpc/interceptor/namespace_validator.go [hunk 2]
```diff
@@ -71,10 +71,30 @@ var (
...
+		"ReplicateEventsV2":                  {},
+		"GetWorkflowExecutionRawHistoryV2":   {},
```
Comment (xwduan) [inline common/rpc/interceptor/namespace_validator.go:78]: `GetWorkflowExecutionRawHistory` api as well? It is a new api built to retrieve events inclusive/inclusive.

**Review by xwduan — APPROVED** [review]

File: common/rpc/interceptor/namespace_validator.go (hunk 1 above)
Comment (yux0) [reply common/rpc/interceptor/namespace_validator.go:76]: Stream is not namespace specific.

File: common/rpc/interceptor/namespace_validator.go (hunk 2 above)
Comment (yux0) [reply common/rpc/interceptor/namespace_validator.go:78]: Have we full move to V2 API?

Comment (xwduan) [reply common/rpc/interceptor/namespace_validator.go:79]: Not yet. The plan was to get rid of V2 api (exclusive/exclusive) and keep inclusive/inclusive api.

**Review by yycptt — APPROVED** [review]

**Pull request merged**